loader.js

```diff
@@ -388,7 +388,8 @@ var Module = null;
 
      function get_mame_files(cfgr, metadata, modulecfg, filelist) {
        var files = [],
-           bios_files = modulecfg['bios_filenames'];
+           bios_files = modulecfg['bios_filenames'],
+           already_fetched_urls = [];
        bios_files.forEach(function (fname, i) {
                             if (fname) {
                               var title = "Bios File ("+ (i+1) +" of "+ bios_files.length +")";
```

Comment:
We should push this url too, for completeness.

Reply:
Yeah, I was wondering about that: lots of combinations this won't catch - and also won't happen, but I'll commit something to add the BIOS files. (We'll double-fetch the CFG file if it's somehow the game file or something, but... this should catch most of the issues.)

Reply:
We’ll just call that “user error” :)